.github/pull_request_template.md

```diff
@@ -36,7 +52,7 @@ _[Add and remove items below accordingly]_
   - [ ] _step2_
 - [ ] Added to [monthly product update](https://www.notion.so/metriport/Customer-Updates-21b4e9d3ad5f4fd68db587a11db28cff?pvs=4) (deprecates a feature that needs to be communicated with customers)
 - [ ] Added to [monthly product update](https://www.notion.so/metriport/Customer-Updates-21b4e9d3ad5f4fd68db587a11db28cff?pvs=4) (introduce a feature that would be useful customers)
-- [ ] Upstream dependencies are met
+- [ ] Upstream dependencies are met/released
 - [ ] Release NPM packages
```

Comment:
maybe add: "DB migrations if relevant"

Reply:
There's already an item about DB migrations, @jonahkaye . Do you mean to add "if relevant" there?

If so, then we would need to add to every item, and there's already a note indicating to add/remove/edit the items as needed.

Reply:
@jonahkaye bump ^